<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNInformation</class>
 <widget class="QWidget" name="SNInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SNInformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(238, 255, 250, 200);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    /* 背景渐变色*/
   /* background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);*/
background-color: rgba(238, 255, 250, 0);
    /* 边框 */
    border: 2px solid/* blue*/;
border-color: rgba(0, 170, 255, 150);

    /* 倒角 */
    border-radius: 8px;

    /* 就像墙上挂着的两个相框,margin指的是相框与相框的距离
       padding指的是每个相框里照片与相框边框的距离 */
    margin-top: 20px;
}</string>
   </property>
   <property name="title">
    <string>SN信息</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>录入信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
